controllers/courses.js

```diff
@@ -169,10 +176,9 @@ const editCourseHandler = (req, res, next) => {
 			s => s.schoolId === res.locals.currentSchool,
 		);
 		const substitutions = _.cloneDeep(
-			teachers.filter(t => t._id !== res.locals.currentUser._id),
+			teachers,
```

Comment:
Could you explain why this filter isn't necessary anymore?

Reply:
this filter will lead to substitution teacher kicking themselves out of the course they edit, because their name will not be included in the substitution list. Still this is not the best solution and was just used for testing.